components/brave_vpn/brave_vpn_os_connection_api_win.cc

```diff
@@ -5,7 +5,10 @@
 
 #include "brave/components/brave_vpn/brave_vpn_os_connection_api_win.h"
 
+// clang-format off
+#include <windows.h>
```

Comment:
why clang-format off?

Reply:
Below order will give compile error. `ras.h` should be put after `windows.h` always.
```
#include <ras.h>
#include <windows.h>
```

Reply:
yes, format wanted to sort them alphabetically, but most of the time `windows.h` needs to be included first.

Reply:
I think you just can add an empty line like this:
```
#include <windows.h>

#include <ras.h>
```

Reply:
Didn't complain locally with this change. Let's see if CI likes it.